extension/bearertokenauth/README.md

```diff
@@ -0,0 +1,40 @@
+# Authenticator - Bearer
+
+This extension implements `configauth.Authenticator` and is to be used in grpc receivers inside the `auth` settings as a means
+to embed a static token for every rpc call that will be made.
+
+The authenticator type has to be set to `bearer`.
+
+## Configuration
+
+```yaml
+extensions:
+   bearertokenauth:
+    token: "sometoken"
+
+receivers:
+  hostmetrics:
+
+exporters:
+  otlp:
+    endpoint: localhost:1234
+    authentication:
```

Comment:
Isn't this here just `auth` now? The examples will need to be tested before merging.